releases/cert-manager/values.yaml.gotmpl

```diff
@@ -0,0 +1,16 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+cert-manager values template file.
+*/}}
+
+## Resources
+resources:
+  requests: {{- .Values | get "_set.resources.requests" dict | toYaml | nindent 4 }}
+  limits: {{- .Values | get "_set.resources.limits" dict | toYaml | nindent 4 }}
+
+## ingressShim (default issuer config)
+{{- if typeIs "string" (.Values._set | get "default_issuer" nil) -}}
+  {{- with .Values._default | get (list .Values._set.default_issuer "ingressShim" | join ".") dict }}
+ingressShim: {{- . | toYaml | nindent 2 }}
```

Comment:
WDYT on adding some more indent just to highlight that it's inside a `with` statement ?

Reply:
@voron Would this be more explicit?

Reply:
I expect something like 
```yaml
{{- if typeIs "string" (.Values._set | get "default_issuer" nil) -}}
  {{- $default_shim := list .Values._set.default_issuer "ingressShim" | join "." -}}
  {{- with .Values._default | get $default_shim dict }}
    ingressShim: {{- . | toYaml | nindent 2 }}
  {{- end  }}
{{- end  }}
```
but then I got that we cannot indent `ingressShim`.